frontend/public/components/overview/project-overview.tsx

```diff
@@ -215,7 +215,7 @@ const ProjectOverviewListItem = connect<ProjectOverviewListItemPropsFromState, P
     // Hide metrics when a selection is active.
     const hasSelection = !!selectedUID;
     const isSelected = uid === selectedUID;
-    const className = classnames('project-overview__item', {'project-overview__item--selected': isSelected});
+    const className = classnames(`project-overview__item project-overview__item--${kind}`, {'project-overview__item--selected': isSelected});
```

Comment:
Should use the full `GroupVersionKind` by calling `referenceFor(obj)` here.

Reply:
I don't think it's a valid CSS classname :/